src/main/java/ImportedDataToCsv.java

```diff
@@ -0,0 +1,23 @@
+import exception.ExecuteQueryException;
+
+import java.sql.Connection;
+import java.sql.Statement;
+
+public class ImportedDataToCsv {
+    public void importData(Connection connection, String path) {
```

Comment:
move Connection to the field, because it looks like util method that can be static